tests/Microsoft.Bot.Builder.Tests/BotFrameworkAdapterTests.cs

```diff
@@ -296,11 +304,19 @@ public async Task ContinueConversationAsyncWithoutAudience()
 
                 var scope = turnContext.TurnState.Get<string>(BotAdapter.OAuthScopeKey);
                 Assert.Equal(AuthenticationConstants.ToChannelFromBotOAuthScope, scope);
+
+                // Ensure the serviceUrl was added to the trusted hosts
+                Assert.True(AppCredentials.TrustedHostNames.ContainsKey(new Uri(channelServiceUrl).Host));
```

Comment:
After searching the tests, it looks like the channelServiceUrl used here may already be cached via an earlier [test](https://github.com/microsoft/botbuilder-dotnet/blob/7b57ecbd2af61abde71d03c5e8189c317f4108b5/tests/Microsoft.Bot.Builder.Tests/BotFrameworkAdapterTests.cs#L176)

I recommend changing the value of channelServiceUrl in this test to a unique value to ensure the test isn't passing due to previously completed tests.

Reply:
Interesting, I added an explicit check on line 318 to ensure the channelServiceUrl is not in the TrustedHostNames before calling continue. But I will change the host URL for this test just to be on the safe side.

Reply:
Updated